lib/core/Schema.js

```diff
@@ -39,7 +39,14 @@ const MutatingWorkflowContextSchema = {
       type: 'object',
       properties: {
         email: { type: 'string' },
-        name: { type: 'string' } // Name is required when creating account
+        name: { type: 'string' }, // Name is required when creating account
+        group: {
```

Comment:
S: Update the typedef `MutatingContextType` JSDoc schema as well to reflect the new JSON schema.